terraform/helm/chaos/templates/ingress.yaml

```diff
@@ -0,0 +1,42 @@
+{{- if .Values.ingress.enable }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
```

Comment:
is that for the chaos mesh dashbaoard?

Reply:
Yep this is for the dashboard. We have to define our own Ingress resource since the one provided by the chaos mesh helm chart does not include things like SSL redirect.